cinema/urls.py

```diff
@@ -18,6 +18,8 @@
 router.register("movie_sessions", MovieSessionViewSet)
 router.register("orders", OrderViewSet)
 
-urlpatterns = [path("", include(router.urls))]
+urlpatterns = [
+    path("", include(router.urls)),
+]
```

Comment:
In case `urlpatterns` consist of only router urls, you could assign `router.urls` to `urlpatterns` directly. Follow this approach in the future